README.md

```diff
@@ -16,66 +16,30 @@ This repo is versioned with [Lerna](https://github.com/lerna/lerna).
-
-While we are getting our `lerna` process ironed out, packages will sometimes not get pulled in correctly. When this happens, auto-rebuild can be broken. If a single package is not found, cd into the directory and run `npm link {package name}`.
-
-## Contributing to Design System
-
-By default, the repo links everything under `src` for local development. Sometimes, development may need to happen under a fixed version of `styles`.
-
-1. Navigate to `storybook`
-2. `Run npm install design-system-styles@[version-number] --save-dev --save-exact`
-3. You may need to remove `node_modules` and/or `package-lock.json`, depending on whether the install is throwing errors
-
-## Publishing
-
-You can publish npm modules from this repo by running:
-`lerna publish` from the main folder.
-Publishing from sub-folders using `npm publish` will result in broken components - don't do this!
-
-First, make sure you are on the `development` branch.
-
-If you have already bumped `package.json`:
-
-    Select the custom version option
-    Re-type the version you've specified in your `package.json`
-    ALWAYS bump the `storybook` package with a patch.
-
-If you have NOT already bumped package.json:
-
-    Select patch option
-    ALWAYS bump the `storybook` package with a patch.
-
 ## Using the Design System
 
-The Design Systems offers modules and styles in different packages. To use modules from the Design System, you need the [styles](https://www.npmjs.com/package/@nypl/design-system-styles) and either [react](https://www.npmjs.com/package/@nypl/design-system-react-components) or [twig](https://www.npmjs.com/package/@nypl/design-system-twig) packages.
-
-### Importing Styles
-
-The design system does not compile styles and templates. It is up to the product to do this. This generally means updating your webpack (grunt, gulp, etc) config. You can use the [storybook config](https://github.com/NYPL/nypl-design-system/blob/development/storybook/.storybook-react/webpack.config.js) as an example, and further documentation in the [styles readme](https://github.com/NYPL/nypl-design-system/blob/development/styles/README.md).
+To use the Design System in your project:
 
-Make sure to include `glob-importer` in your (sass-loader)[https://github.com/NYPL/nypl-design-system/blob/development/storybook/.storybook-react/webpack.config.js#L2].
+1. Install the design system `npm install @nypl/design-system-react-components`
+2. Import the styles in your main css file `@import '~@nypl/design-system-react-components/dist/app.css';`
+3. Consult Storybook for the list of available components and props that they require.
+4. Usage example
 
-### Importing React Components
+```
+import * as DS from '@nypl/design-system-react-components';
```

Comment:
I may be wrong, but would importing this way make tree shaking impossible? As in, if we `import * as DS` instead of `import { Link } from`, will we always be including the whole bundle, even if we only use one component?

Reply:
To be honest, I haven't given much thought to tree shaking.  Looking at it in webpack, it seems like there's a some other things that I'll have to check in this library to be able to label it with `"sideEffects": false` so it's currently not being treeshaken at all.  

There's also not currently (and there are no plans to) support for something like 
`import { Heading, Link } as DS` which means that we would have to do something like 
`import { Heading as DsHeading, Link as DsLink }` (the aliasing is important because you can imagine, something like `Link` is going to be easily confused with `ReactRouter.Link` )`

I think this might need a whole other thought process around it (perhaps if/when we migrate to Rollup which supports treeshaking much more out-of-box).  

In the meantime, I will update the readme to use `import { Link } from` but add a note that people might need aliases.